config/default.yaml

```diff
@@ -229,6 +229,7 @@ contact_form:
 signup:
   enabled: false
   limit: 10 # When the limit is reached, registrations are disabled. -1 == unlimited
+  minimum_age: 16
```

Comment:
Please add a comment to explain this is used in the signup form

Reply:
```suggestion
  minimum_age: 16 # Used to configure the signup form
```